docs/tutorial/part-seven/index.md

```diff
@@ -254,35 +251,32 @@ exports.onCreateNode = ({ node, getNode, actions }) => {
 
 exports.createPages = ({ graphql, actions }) => {
   const { createPage } = actions // highlight-line
-  return new Promise((resolve, reject) => {
-    graphql(`
-      {
-        allMarkdownRemark {
-          edges {
-            node {
-              fields {
-                slug
-              }
+  return graphql(`
```

Comment:
Same idea down here. I don't have as strong of an opinion as to whether we need it in both places.